*** /tmp/dphQoaa	Mon Jul 30 09:27:18 2012
--- /tmp/bp_check.auto.757776	Mon Jul 30 09:27:18 2012
***************
*** 95,96 ****
--- 95,97 ----
  HELP  128.40.163.206  -> 144.82.104.1    uclinst-gw1.uclinst
+ HELP  128.40.163.30   -> 144.82.103.129  uclusers-slp2.uclusers
  HELP  128.40.163.30   -> 144.82.104.20   uclinst-gw20.uclinst
***************
*** 379,380 ****
--- 380,382 ----
  HELP  144.82.117.126  -> 144.82.117.40   ucl17-gw1.ucl17
+ HELP  144.82.117.254  -> 144.82.117.40   ucl17-gw1.ucl17
  HELP  144.82.119.245  -> 144.82.103.3    uclclust-lp3.uclclust
