*** /tmp/d5-mjaa	Thu Sep 14 16:33:50 2006
--- /tmp/bp_check.auto.1061050	Thu Sep 14 16:33:50 2006
***************
*** 1,2 ****
- NAME  arts-ws3                <not in DNS>    00188B053D5B  uclinst-gw23.uclinst
  NAME  cee-ws20.cee            <not in DNS>    0080C857967D  uclinst-gw19.uclinst
--- 1 ----
***************
*** 4,8 ****
  NAME  crucilib-ws19.crucilib  <not in DNS>    XXXXXXXXXXXX  uclinst-gw5.uclinst
- NAME  histart-ws26            <not in DNS>    XXXXXXXXXXXX  uclinst-gw23.uclinst
- NAME  histart-ws31            <not in DNS>    XXXXXXXXXXXX  uclinst-gw23.uclinst
- NAME  histart-ws33            <not in DNS>    XXXXXXXXXXXX  uclinst-gw23.uclinst
  NAME  laws-ws11.laws          <not in DNS>    XXXXXXXXXXXX  uclinst-gw22.uclinst
--- 3,4 ----
***************
*** 103,108 ****
  MAC   XXXXXXXXXXXX  hist-ws8.hist           128.40.88.95    uclinst-gw14.uclinst
! MAC   XXXXXXXXXXXX  histart-ws26            <not in DNS>    uclinst-gw23.uclinst
! MAC   XXXXXXXXXXXX  histart-ws31            <not in DNS>    uclinst-gw23.uclinst
  MAC   XXXXXXXXXXXX  histart-ws32.histart    128.40.187.32   uclinst-gw16.uclinst
! MAC   XXXXXXXXXXXX  histart-ws33            <not in DNS>    uclinst-gw23.uclinst
  MAC   XXXXXXXXXXXX  iseisd-ws102.is-eisd    128.40.222.52   uclinst-gw20.uclinst
--- 99,104 ----
  MAC   XXXXXXXXXXXX  hist-ws8.hist           128.40.88.95    uclinst-gw14.uclinst
! MAC   XXXXXXXXXXXX  histart-ws26.histart    128.40.187.26   uclinst-gw23.uclinst
! MAC   XXXXXXXXXXXX  histart-ws31.histart    128.40.187.31   uclinst-gw23.uclinst
  MAC   XXXXXXXXXXXX  histart-ws32.histart    128.40.187.32   uclinst-gw16.uclinst
! MAC   XXXXXXXXXXXX  histart-ws33.histart    128.40.187.33   uclinst-gw23.uclinst
  MAC   XXXXXXXXXXXX  iseisd-ws102.is-eisd    128.40.222.52   uclinst-gw20.uclinst
