*** /tmp/dPKajMa	Fri Jul 24 09:09:48 2015
--- /tmp/bp_check.auto.3735772	Fri Jul 24 09:09:47 2015
***************
*** 353,355 ****
  HELP  193.60.234.190  -> 144.82.104.10   uclinst-gw10.uclinst
- HELP  193.60.242.245  -> 144.82.103.129  uclusers-slp2.uclusers
  HELP  193.60.242.245  -> 144.82.104.10   uclinst-gw10.uclinst
--- 353,354 ----
***************
*** 364,367 ****
  HELP  194.66.92.245   -> 144.82.104.19   uclinst-gw19.uclinst
- HELP  195.195.96.245  -> 10.50.58.7      slmsdhcp01.ad.ucl.ac.uk
- HELP  195.195.96.245  -> 10.50.58.8      slmsdhcp02.ad.ucl.ac.uk
  HELP  195.195.96.245  -> 144.82.104.19   uclinst-gw19.uclinst
--- 363,364 ----
