*** /tmp/dllidaa	Wed Aug 14 10:17:19 2013
--- /tmp/bp_check.auto.745556	Wed Aug 14 10:17:19 2013
***************
***************
*** 23,25 ****
  HELP  128.40.127.245  -> 144.82.104.15   uclinst-gw15.uclinst
- HELP  128.40.128.245  -> 144.82.103.1    uclclust-lp1.uclclust
  HELP  128.40.128.245  -> 144.82.103.3    uclclust-lp3.uclclust
--- 24,25 ----
***************
*** 125,128 ****
  HELP  128.40.174.245  -> 144.82.104.17   uclinst-gw17.uclinst
- HELP  128.40.175.245  -> 144.82.103.128  uclusers-slp1.uclusers
- HELP  128.40.175.245  -> 144.82.104.5    uclinst-gw5.uclinst
  HELP  128.40.176.245  -> 144.82.103.131  uclusers-slp4.uclusers
--- 125,126 ----
***************
*** 328,330 ****
  HELP  128.40.60.245   -> 144.82.104.8    uclinst-gw8.uclinst
- HELP  128.40.62.245   -> 144.82.104.15   uclinst-gw15.uclinst
  HELP  128.40.63.245   -> 144.82.103.1    uclclust-lp1.uclclust
--- 326,327 ----
***************
*** 373,375 ****
  HELP  128.40.89.238   -> 144.82.104.22   uclinst-gw22.uclinst
- HELP  128.40.89.30    -> 144.82.103.1    uclclust-lp1.uclclust
  HELP  128.40.89.30    -> 144.82.104.20   uclinst-gw20.uclinst
--- 370,371 ----
