*** /tmp/dxSal7a	Wed Mar 23 12:43:38 2016
--- /tmp/bp_check.auto.10420258	Wed Mar 23 12:43:38 2016
***************
*** 13,16 ****
  HELP  128.40.116.254  -> 144.82.104.22   uclinst-gw22.uclinst
  HELP  128.40.118.126  -> 144.82.104.16   uclinst-gw16.uclinst
- HELP  128.40.118.126  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.121.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 13,16 ----
  HELP  128.40.116.254  -> 144.82.104.22   uclinst-gw22.uclinst
+ HELP  128.40.118.126  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.118.126  -> 144.82.104.16   uclinst-gw16.uclinst
  HELP  128.40.121.254  -> 144.82.103.128  uclusers-slp1.uclusers
***************
*** 50,51 ****
--- 50,52 ----
  HELP  128.40.148.245  -> 144.82.104.21   uclinst-gw21.uclinst
+ HELP  128.40.15.254   -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.15.254   -> 144.82.104.11   uclinst-gw11.uclinst
***************
*** 52,54 ****
  HELP  128.40.15.254   -> 144.82.104.7    uclinst-gw7.uclinst
- HELP  128.40.15.254   -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.150.126  -> 144.82.104.4    uclinst-gw4.uclinst
--- 53,54 ----
***************
*** 140,141 ****
--- 140,142 ----
  HELP  128.40.196.245  -> 144.82.104.20   uclinst-gw20.uclinst
+ HELP  128.40.198.254  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.198.254  -> 144.82.103.128  uclusers-slp1.uclusers
***************
*** 143,145 ****
  HELP  128.40.198.254  -> 144.82.104.5    uclinst-gw5.uclinst
- HELP  128.40.198.254  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.2.245    -> 144.82.104.23   uclinst-gw23.uclinst
--- 144,145 ----
***************
*** 169,170 ****
--- 169,171 ----
  HELP  128.40.218.245  -> 144.82.104.22   uclinst-gw22.uclinst
+ HELP  128.40.223.254  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.223.254  -> 144.82.103.128  uclusers-slp1.uclusers
***************
*** 171,173 ****
  HELP  128.40.223.254  -> 144.82.104.20   uclinst-gw20.uclinst
- HELP  128.40.223.254  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.228.245  -> 144.82.103.1    uclclust-lp1.uclclust
--- 172,173 ----
***************
*** 264,266 ****
  HELP  128.40.63.245   -> 144.82.104.23   uclinst-gw23.uclinst
! HELP  128.40.65.126   -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
  HELP  128.40.68.126   -> 144.82.103.131  uclusers-slp4.uclusers
--- 264,267 ----
  HELP  128.40.63.245   -> 144.82.104.23   uclinst-gw23.uclinst
! HELP  128.40.65.126   -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.40.65.126   -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
  HELP  128.40.68.126   -> 144.82.103.131  uclusers-slp4.uclusers
***************
*** 307,310 ****
  HELP  128.40.98.245   -> 144.82.104.8    uclinst-gw8.uclinst
! HELP  128.41.7.254    -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
  HELP  144.82.117.126  -> 144.82.104.20   uclinst-gw20.uclinst
--- 308,313 ----
  HELP  128.40.98.245   -> 144.82.104.8    uclinst-gw8.uclinst
! HELP  128.41.7.254    -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.41.7.254    -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
  HELP  144.82.117.126  -> 144.82.104.20   uclinst-gw20.uclinst
