*** /tmp/d0lqnya	Wed Nov 15 11:00:47 2006
--- /tmp/bp_check.auto.495636	Wed Nov 15 11:00:47 2006
***************
*** 51,55 ****
  MAC   XXXXXXXXXXXX  geog-ws158.geog         128.40.32.31    uclinst-gw19.uclinst
- MAC   XXXXXXXXXXXX  geog-ws60.geog          128.40.32.123   uclinst-gw19.uclinst
- MAC   XXXXXXXXXXXX  geog-ws61.geog          128.40.32.124   uclinst-gw19.uclinst
- MAC   XXXXXXXXXXXX  geog-ws63.geog          128.40.32.126   uclinst-gw19.uclinst
  MAC   XXXXXXXXXXXX  german-ws20.german      128.40.191.186  uclinst-gw22.uclinst
--- 51,52 ----
