*** /tmp/dY-ifaa	Thu Aug 29 14:48:03 2013
--- /tmp/bp_check.auto.696324	Thu Aug 29 14:48:03 2013
***************
*** 98,100 ****
  HELP  128.40.162.245  -> 144.82.104.11   uclinst-gw11.uclinst
- HELP  128.40.162.245  -> 144.82.104.19   uclinst-gw19.uclinst
  HELP  128.40.162.245  -> 144.82.104.23   uclinst-gw23.uclinst
--- 98,99 ----
***************
*** 146,148 ****
  HELP  128.40.184.238  -> 144.82.104.22   uclinst-gw22.uclinst
- HELP  128.40.184.238  -> 144.82.104.5    uclinst-gw5.uclinst
  HELP  128.40.184.246  -> 144.82.104.7    uclinst-gw7.uclinst
--- 145,146 ----
