*** /tmp/d5nyfqa	Fri Dec  7 13:31:56 2012
--- /tmp/bp_check.auto.999576	Fri Dec  7 13:31:56 2012
***************
*** 162,163 ****
--- 162,164 ----
  HELP  128.40.190.126  -> 144.82.104.11   uclinst-gw11.uclinst
+ HELP  128.40.190.14   -> 144.82.104.3    uclinst-gw3.uclinst
  HELP  128.40.190.222  -> 144.82.104.23   uclinst-gw23.uclinst
***************
*** 218,219 ****
--- 219,221 ----
  HELP  128.40.223.254  -> 144.82.104.20   uclinst-gw20.uclinst
+ HELP  128.40.223.254  -> 144.82.117.40   ucl17-gw1.ucl17
  HELP  128.40.228.245  -> 144.82.103.3    uclclust-lp3.uclclust
