*** /tmp/d-Tagya	Wed Nov  9 10:15:12 2016
--- /tmp/bp_check.auto.6946980	Wed Nov  9 10:15:11 2016
***************
*** 11,14 ****
  HELP  128.40.116.254  -> 144.82.104.22   uclinst-gw22.uclinst
- HELP  128.40.118.126  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.118.126  -> 144.82.104.16   uclinst-gw16.uclinst
  HELP  128.40.121.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 11,14 ----
  HELP  128.40.116.254  -> 144.82.104.22   uclinst-gw22.uclinst
  HELP  128.40.118.126  -> 144.82.104.16   uclinst-gw16.uclinst
+ HELP  128.40.118.126  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.121.254  -> 144.82.103.128  uclusers-slp1.uclusers
***************
*** 44,46 ****
  HELP  128.40.148.245  -> 144.82.104.21   uclinst-gw21.uclinst
- HELP  128.40.15.254   -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.15.254   -> 144.82.104.11   uclinst-gw11.uclinst
--- 44,45 ----
***************
*** 47,48 ****
--- 46,48 ----
  HELP  128.40.15.254   -> 144.82.104.7    uclinst-gw7.uclinst
+ HELP  128.40.15.254   -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.150.126  -> 144.82.104.4    uclinst-gw4.uclinst
***************
*** 129,131 ****
  HELP  128.40.196.245  -> 144.82.104.20   uclinst-gw20.uclinst
- HELP  128.40.198.254  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.198.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 129,130 ----
***************
*** 133,134 ****
--- 132,134 ----
  HELP  128.40.198.254  -> 144.82.104.5    uclinst-gw5.uclinst
+ HELP  128.40.198.254  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.2.245    -> 144.82.104.23   uclinst-gw23.uclinst
***************
*** 155,157 ****
  HELP  128.40.218.245  -> 144.82.104.22   uclinst-gw22.uclinst
- HELP  128.40.223.254  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.223.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 155,156 ----
***************
*** 158,159 ****
--- 157,159 ----
  HELP  128.40.223.254  -> 144.82.104.20   uclinst-gw20.uclinst
+ HELP  128.40.223.254  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.228.245  -> 144.82.103.1    uclclust-lp1.uclclust
***************
*** 249,252 ****
  HELP  128.40.63.245   -> 144.82.104.23   uclinst-gw23.uclinst
! HELP  128.40.65.126   -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.40.65.126   -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
  HELP  128.40.68.126   -> 144.82.103.131  uclusers-slp4.uclusers
--- 249,251 ----
  HELP  128.40.63.245   -> 144.82.104.23   uclinst-gw23.uclinst
! HELP  128.40.65.126   -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
  HELP  128.40.68.126   -> 144.82.103.131  uclusers-slp4.uclusers
***************
*** 287,292 ****
  HELP  128.40.98.245   -> 144.82.104.8    uclinst-gw8.uclinst
! HELP  128.41.7.254    -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.41.7.254    -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
  HELP  144.82.117.126  -> 144.82.104.20   uclinst-gw20.uclinst
--- 286,289 ----
  HELP  128.40.98.245   -> 144.82.104.8    uclinst-gw8.uclinst
! HELP  128.41.7.254    -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
  HELP  144.82.117.126  -> 144.82.104.20   uclinst-gw20.uclinst
