*** /tmp/dQZmaUa	Tue Feb  6 13:39:50 2007
--- /tmp/bp_check.auto.639108	Tue Feb  6 13:39:50 2007
***************
*** 106,108 ****
--- 109,113 ----
  MAC   XXXXXXXXXXXX  npsk.cloist             128.40.170.46   uclusers-slp4.uclusers
+ MAC   XXXXXXXXXXXX  npsk.psychiat           128.40.240.128  uclusers-slp3.uclusers
  MAC   XXXXXXXXXXXX  npsl.cloist             128.40.170.47   uclusers-slp4.uclusers
+ MAC   XXXXXXXXXXXX  npsl.psychiat           128.40.240.129  uclusers-slp3.uclusers
  MAC   XXXXXXXXXXXX  npsn.med                128.40.253.165  uclusers-slp3.uclusers
***************
*** 109,110 ****
--- 114,116 ----
  MAC   XXXXXXXXXXXX  npsn.sopps              128.40.176.33   uclusers-slp4.uclusers
+ MAC   XXXXXXXXXXXX  npss.is-eisd            128.40.222.38   uclusers-slp1.uclusers
  MAC   XXXXXXXXXXXX  npsu.is-eisd            128.40.222.40   uclusers-slp1.uclusers
***************
*** 119,120 ****
--- 125,127 ----
  MAC   XXXXXXXXXXXX  studuni-ws66.studuni    193.60.230.216  uclinst-gw10.uclinst
+ MAC   XXXXXXXXXXXX  surg-ws11.surg          128.40.251.122  uclinst-gw8.uclinst
  MAC   XXXXXXXXXXXX  ucl12-ws22.ucl12        128.40.161.22   uclinst-gw22.uclinst
***************
*** 526,529 ****
  HELP  128.40.98.245   -> 144.82.104.8    uclinst-gw8.uclinst
- HELP  144.82.117.126  -> 144.82.117.24   agwa.ucl17
- HELP  144.82.117.245  -> 144.82.117.43   ucl17-gw7.ucl17
  HELP  144.82.119.245  -> 144.82.103.3    uclclust-lp3.uclclust
--- 533,534 ----
