*** /tmp/dq_upUa	Tue Nov  7 14:38:59 2006
--- /tmp/bp_check.auto.1278126	Tue Nov  7 14:38:59 2006
***************
*** 123,125 ****
  MAC   XXXXXXXXXXXX  npsn.sopps              128.40.176.33   uclusers-slp4.uclusers
- MAC   XXXXXXXXXXXX  npss.is-eisd            128.40.222.38   uclusers-slp1.uclusers
  MAC   XXXXXXXXXXXX  npsu.is-eisd            128.40.222.40   uclusers-slp1.uclusers
--- 123,124 ----
