*** /tmp/dJ_ajaa	Fri Aug  8 09:46:43 2014
--- /tmp/bp_check.auto.7602204	Fri Aug  8 09:46:43 2014
***************
*** 74,76 ****
  HELP  128.40.161.30   -> 144.82.104.22   uclinst-gw22.uclinst
- HELP  128.40.162.245  -> 144.82.104.11   uclinst-gw11.uclinst
  HELP  128.40.162.245  -> 144.82.104.23   uclinst-gw23.uclinst
--- 74,75 ----
***************
*** 196,198 ****
  HELP  128.40.228.245  -> 144.82.104.5    uclinst-gw5.uclinst
- HELP  128.40.23.254   -> 144.82.103.129  uclusers-slp2.uclusers
  HELP  128.40.23.254   -> 144.82.104.19   uclinst-gw19.uclinst
--- 195,196 ----
