*** /tmp/d--Udia	Thu Jan 24 12:18:40 2008
--- /tmp/bp_check.auto.3641540	Thu Jan 24 12:18:40 2008
***************
*** 65,67 ****
  HELP  128.40.139.245  -> 144.82.103.2    uclclust-lp2.uclclust
- HELP  128.40.139.245  -> 144.82.104.6    uclinst-gw6.uclinst
  HELP  128.40.140.245  -> 144.82.100.187  cbds-a.ucl0
--- 65,66 ----
***************
*** 395,396 ****
--- 394,396 ----
  HELP  128.40.89.62    -> 144.82.104.22   uclinst-gw22.uclinst
+ HELP  128.40.89.78    -> 144.82.104.20   uclinst-gw20.uclinst
  HELP  128.40.89.78    -> 144.82.104.4    uclinst-gw4.uclinst
