*** /tmp/dW4yd7a	Fri Mar 11 08:39:06 2011
--- /tmp/bp_check.auto.1298446	Fri Mar 11 08:39:06 2011
***************
*** 295,297 ****
  HELP  128.40.48.158   -> 144.82.103.4    uclclust-lp4.uclclust
! HELP  128.40.48.158   -> 144.82.104.10   uclinst-gw10.uclinst
  HELP  128.40.48.158   -> 144.82.104.20   uclinst-gw20.uclinst
--- 295,297 ----
  HELP  128.40.48.158   -> 144.82.103.4    uclclust-lp4.uclclust
! HELP  128.40.48.158   -> 144.82.104.19   uclinst-gw19.uclinst
  HELP  128.40.48.158   -> 144.82.104.20   uclinst-gw20.uclinst
