*** /tmp/d-Lac7a	Tue Jul 22 16:52:22 2014
--- /tmp/bp_check.auto.15532066	Tue Jul 22 16:52:22 2014
***************
*** 1 ****
--- 1,2 ----
+ NAME  nps-a.medsch.ucl.ac.uk  <not in DNS>    0030C15F6CE3  uclclust-lp1.uclclust
  HELP  128.40.101.254  -> 144.82.117.120  bccusers-gw3.bccusers
***************
*** 12,15 ****
  HELP  128.40.117.245  -> 144.82.103.1    uclclust-lp1.uclclust
- HELP  128.40.118.126  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.118.126  -> 144.82.104.16   uclinst-gw16.uclinst
  HELP  128.40.121.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 13,16 ----
  HELP  128.40.117.245  -> 144.82.103.1    uclclust-lp1.uclclust
  HELP  128.40.118.126  -> 144.82.104.16   uclinst-gw16.uclinst
+ HELP  128.40.118.126  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.121.254  -> 144.82.103.128  uclusers-slp1.uclusers
***************
*** 54,56 ****
  HELP  128.40.148.245  -> 144.82.104.21   uclinst-gw21.uclinst
- HELP  128.40.15.254   -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.15.254   -> 144.82.104.11   uclinst-gw11.uclinst
--- 55,56 ----
***************
*** 57,58 ****
--- 57,59 ----
  HELP  128.40.15.254   -> 144.82.104.7    uclinst-gw7.uclinst
+ HELP  128.40.15.254   -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.150.126  -> 144.82.104.4    uclinst-gw4.uclinst
***************
*** 166,168 ****
  HELP  128.40.196.245  -> 144.82.104.20   uclinst-gw20.uclinst
- HELP  128.40.198.254  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.198.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 167,168 ----
***************
*** 170,171 ****
--- 170,172 ----
  HELP  128.40.198.254  -> 144.82.104.5    uclinst-gw5.uclinst
+ HELP  128.40.198.254  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.2.245    -> 144.82.104.23   uclinst-gw23.uclinst
***************
*** 195,197 ****
  HELP  128.40.218.245  -> 144.82.104.22   uclinst-gw22.uclinst
- HELP  128.40.223.254  -> 10.10.115.7     udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.223.254  -> 144.82.103.128  uclusers-slp1.uclusers
--- 196,197 ----
***************
*** 198,199 ****
--- 198,200 ----
  HELP  128.40.223.254  -> 144.82.104.20   uclinst-gw20.uclinst
+ HELP  128.40.223.254  -> <not in DNS>    udxpvs01d.addev.ucl.ac.uk
  HELP  128.40.228.245  -> 144.82.103.1    uclclust-lp1.uclclust
***************
*** 296,299 ****
  HELP  128.40.63.245   -> 144.82.104.23   uclinst-gw23.uclinst
! HELP  128.40.65.126   -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.40.65.126   -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
  HELP  128.40.66.245   -> 144.82.103.129  uclusers-slp2.uclusers
--- 297,299 ----
  HELP  128.40.63.245   -> 144.82.104.23   uclinst-gw23.uclinst
! HELP  128.40.65.126   -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
  HELP  128.40.66.245   -> 144.82.103.129  uclusers-slp2.uclusers
***************
*** 349,354 ****
  HELP  128.41.4.254    -> 144.82.104.21   uclinst-gw21.uclinst
! HELP  128.41.7.254    -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.41.7.254    -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> 10.60.64.61     uddhcp01t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> 10.60.65.61     uddhcp02t.adtest.bcc.ac.uk
  HELP  144.82.117.126  -> 144.82.104.20   uclinst-gw20.uclinst
--- 349,352 ----
  HELP  128.41.4.254    -> 144.82.104.21   uclinst-gw21.uclinst
! HELP  128.41.7.254    -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
! HELP  128.41.8.254    -> <not in DNS>    uddhcp02t.adtest.bcc.ac.uk
  HELP  144.82.117.126  -> 144.82.104.20   uclinst-gw20.uclinst
