*** /tmp/dS_Yd7a	Mon Jan 10 16:48:04 2011
--- /tmp/bp_check.auto.1101986	Mon Jan 10 16:48:04 2011
***************
*** 1 ****
--- 1,3 ----
+ MAC   00215A99C293  npsao.is-eisd           128.40.210.152  uclusers-slp1.uclusers
+ MAC   00215A99C293  npso.is-eisd            128.40.222.34   uclusers-slp1.uclusers
  HELP  128.40.108.245  -> 144.82.103.4    uclclust-lp4.uclclust
***************
*** 269,270 ****
--- 271,273 ----
  HELP  128.40.34.245   -> 144.82.104.23   uclinst-gw23.uclinst
+ HELP  128.40.37.126   -> 144.82.104.7    uclinst-gw7.uclinst
  HELP  128.40.37.174   -> 144.82.104.17   uclinst-gw17.uclinst
