*** /tmp/dXx3jaa	Thu Mar 29 13:35:58 2007
--- /tmp/bp_check.auto.1663124	Thu Mar 29 13:35:58 2007
***************
*** 20,23 ****
  MAC   XXXXXXXXXXXX  cbss-ws11.cbss          128.40.240.171  uclinst-gw21.uclinst
- MAC   XXXXXXXXXXXX  cbss-ws7.cbss           128.40.240.167  uclinst-gw21.uclinst
- MAC   XXXXXXXXXXXX  cbss-ws8.cbss           128.40.240.168  uclinst-gw21.uclinst
  MAC   XXXXXXXXXXXX  cbss-ws9.cbss           128.40.240.169  uclinst-gw21.uclinst
--- 20,21 ----
